backend-issuer/src/env.ts

```diff
@@ -11,14 +11,18 @@ export default envalid.cleanEnv(process.env, {
   PORT: envalid.port({ default: 3000 }),
   LOG_LEVEL: envalid.str({ default: 'info', devDefault: 'debug' }),
   CLOUDAGENT_HOST: envalid.host({ devDefault: 'localhost' }),
-  CLOUDAGENT_PORT: envalid.port({ default: 3000, devDefault: 3001 }),
-  DB_HOST: envalid.str({ devDefault: 'localhost' }),
+  CLOUDAGENT_PORT: envalid.port({ default: 3000, devDefault: 3010 }),
+  DB_HOST: envalid.str({ default: 'postgres', devDefault: 'localhost' }),
   DB_PORT: envalid.port({ default: 5432 }),
-  DB_USERNAME: envalid.str({ devDefault: 'postgres' }),
-  DB_PASSWORD: envalid.str({ devDefault: 'postgres' }),
+  DB_USERNAME: envalid.str({ default: 'postgres' }),
+  DB_PASSWORD: envalid.str({ default: 'postgres' }),
   DB_NAME: envalid.str({ default: 'nice-agent-portal' }),
   STATIC_ROOT: envalid.str({
     default: '/www',
     devDefault: '../frontend/build',
   }),
+  NICE_AGENT_ROLE: envalid.str({
+    choices: ['peer', 'issuer'],
+    default: 'peer',
+  }),
```

Comment:
is `NICE_AGENT_ROLE` required in this file, looks like it's only used at build time rather than runtime?